tests/views/test_describe_image.py

```diff
@@ -133,3 +134,45 @@ def test_custom_rendition_filter(admin_client, settings):
     assert response.json() == {
         "message": "This is an echo backend: images/example.max-100x100.jpg"
     }
+
+
+@pytest.mark.parametrize(
+    "maxlength,expected_status,error_message",
+    [
+        (100, HTTPStatus.OK, None),
+        (
+            -1,
+            HTTPStatus.BAD_REQUEST,
+            "Ensure this value is greater than or equal to 0.",
+        ),
+        (
+            10000,
+            HTTPStatus.BAD_REQUEST,
+            "Ensure this value is less than or equal to 4096.",
+        ),
+    ],
+)
```

Comment:
**Praise:** Love the table-driven tests. 🥰

Reply:
Yeah, I've been reaching for `parametrize` a lot lately, it's comforting that the logic is not duplicated across several tests, which might get out of sync. Pytest is _so_ good. `<3`

Reply:
It's just syntactic sugar I guess. I use [subTest](https://docs.python.org/3/library/unittest.html#unittest.TestCase.subTest) a lot otherwise. Nothing stops you from implementing table-driven tests with whatever tool you have with full honesty..